docs/resource_sets.rst

```diff
@@ -0,0 +1,86 @@
+**********************************
+Resource sets and partial compile
+**********************************
+To reduce the time of a compile action, resource sets can be used. Without the use of those, each time a resource is added,
```

Comment:
`To reduce the time of a compile action`, how about just compile time? `In order to reduce the compile time`.